### DISCUSSION TITLE
Library filtering

### DISCUSSION THREAD

**Discussion by heyhippari** [jellyfin/jellyfin-vue#145] [Features]

**Comment by AlfHou** [comment]
I'll take a look at this

**Comment by AlfHou** [comment]
I'm looking at the network calls the jellyfin-web client makes when selecting filters. My understanding is that the filtering is done serverside. Is that correct?

**Comment by heyhippari** [comment]
This is correct.

We have an API client available through `this.$api` all around the project, which you can use to make requests.
